parkhangs-webapp/src/features/modal/ModalMapDetail.js

```diff
@@ -108,12 +115,23 @@ class ModalMapDetail extends React.Component {
                                         <td>
                                             <b>Delete</b>
                                         </td>
+                                        <td>
+                                            <b>Add To GCalendar</b>
+                                        </td>
                                     </tr>
                                     </thead>
                                     <tbody>
                                     {
 
                                         this.getEventsByPark().map((event) => {
+                                            
+                                            let newE = {
+                                                title: event.details,
+                                                description: event.details,
+                                                location: this.props.park.name + " BC, Canada",
```

Comment:
for location do you mean address? if so, `this.props.park.streetNumber + " " + this.props.park.streetName`

Reply:
I did that initially but it’s clearer to go directly to the park, since it redirects to google calendar and the picker can identify the park. Both did the same thing from when I tested it.

Reply:
It's a simple change if it bothers you